cmd/wait/runner.go

```diff
@@ -156,6 +163,17 @@ func (r *runner) run(ctx context.Context, _ *cobra.Command, _ []string) error {
 				r.logger.LogCtx(ctx, "message", "error listing services", "error", err)
 				return microerror.Mask(err)
 			}
+			if len(services.Items) == 0 {
+				serviceLabelSelector := labelsToSelector(alternateTargetLabels)
+				services, err = k8sClient.CoreV1().Services("kube-system").List(ctx, v1.ListOptions{
+					LabelSelector: serviceLabelSelector,
+				})
+				if err != nil {
+					r.logger.LogCtx(ctx, "message", "error listing services", "error", err)
+					return microerror.Mask(err)
+				}
+			}
+
 			if len(services.Items) == 0 {
 				message := fmt.Sprintf("CoreDNS service not found using label selector %#q", serviceLabelSelector)
```

Comment:
This can be confusing I guess - we now check with two sets of labels.

Reply:
what alternative do you suggest? there is no single set of labels that can detect both scenarios

Reply:
```
message := fmt.Sprintf("CoreDNS service not found using label selectors %#q or %#q", serviceLabelSelector, alternateTargetLabels)
```

Reply:
I couldn't do a code suggestion for some reason

Reply:
I didn't understand you were referring to the log msg, sorry

Reply:
done